Update NEWS (#2305)
systemd/mkosi: NEWS.md

  building multiple images.
- We now create the `/etc/mtab` compatibility symlink in mkosi's
  sandbox.
- We now always hash the root password ourselves instead of leaving it

>   to systemd-firstboot.
> - /srv and /mnt are not mounted read-only anymore during builds.
> - Fixed a crash when running mkosi in a directory with fewer than two
>   parent directories.

Reviewer: ```suggestion
  to `systemd-firstboot`.
- `/srv` and `/mnt` are not mounted read-only anymore during builds.
- Fixed a crash when running mkosi in a directory whose direct parent is `/`.
```
